### PR TITLE
New export page for project

### PR DESCRIPTION
**Pull request by terrymyc**
This PR redesigned the export page of the project page and removed the publication zone and the export dialog. The feature to mark a project as finished will be placed on the details page.

image

image

### CODE REVIEW

**Comment by Rensvandeschoot** [comment]
Super smart description!!! I like it very much

File: asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+                </MenuItem>
+                <MenuItem value="project">
+                  <Box>
+                    <Typography variant="subtitle1">Project</Typography>
+                    <Typography
+                      variant="body2"
+                      gutterBottom
+                      sx={{ color: "text.secondary" }}
+                    >
+                      Including data and model configuration
```
Comment (J535D165) [inline asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js:155]: @SagevdBrand @Rensvandeschoot Can you think about the wording?

**Review by J535D165 — APPROVED** [review]

**Comment by J535D165** [comment]
Thanks, looking forward to the user feedback.

File: asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js (hunk 1 above)
Comment (SagevdBrand) [reply asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js:155]: Including all information on model settings?

**Pull request merged**